legend-engine-xt-persistence-grammar/src/main/java/org/finos/legend/engine/language/pure/dsl/persistence/grammar/to/PersistenceComposerExtension.java

```diff
@@ -113,6 +122,37 @@ else if (trigger instanceof CronTrigger)
                 });
     }
 
+    @Override
+    public List<Function2<TestAssertion, PureGrammarComposerContext, ContentWithType>> getExtraTestAssertionComposers()
```

Comment:
We should definetly add tests for parser/composer/compiler changes

Reply:
Sure, edited existing tests and added new test cases